README.md

```diff
@@ -16,18 +16,19 @@ _A part of the [greater Fides ecosystem](https://github.com/ethyca/fides)._
 ## :rocket: Quick Start
 If you're looking for a more detailed introduction to fidesops, we recommend following [our tutorial here](https://ethyca.github.io/fidesops/tutorial/). 
 
-Run the quickstart in your terminal to give fidesops a test drive:
+### System Requirements
 
-```
-Install Docker: https://docs.docker.com/desktop/#download-and-install
-Install Make: brew install make
-```
+- [Docker](https://www.docker.com/products/docker-desktop) (20.10.8+)
+- [Python](https://www.python.org/downloads/) (3.8+)
```

Comment:
need to install `nox` here as well

Reply:
@ThomasLaPiana this is part of dev requirement so no need for another install right?

Reply:
yeah as long as the user is installing the dev-requirements, but given this is the README I'm assuming this is for end-users? In which case for the fidesctl README we actually direct users to [install nox with pipx](https://github.com/ethyca/fides#system-requirements) as well

Reply:
Maybe we should direct them to install all requirements here instead then? `pip install -r dev-requirements.txt -r requirements.txt`

With just `pip install nox`, nox will work but the commands it runs will fail, ie `nox -s pytest_integration` wouldn't work.

Reply:
Wait, maybe I'm wrong here since the nox commands would run in docker?

Reply:
I originally didn't consider that anyone would run things without first pip installing all the requirements. I just confirmed it does work with just a `pip install nox` since things run in docker. `pip install nox` after already doing a pip install of the requirements wouldn't hurt anything so with that it wouldn't hurt to put the instruction to install nox back. Sorry for the confusion @conceptualshark.